development/prowjobparser/README.md

```diff
@@ -0,0 +1,20 @@
+# prowjobparser
+
+## Overview
+
+The prowjobparser is a helper tool which parses all ProwJobs under the provided path, matches them against the provided label filters, and prints matching ProwJob names to the standard output.
+
+## Usage
+
+ `~/go/src/github.com/dekiel/test-infra/development/prowjobparser$ go run main.go -c ../../../../kyma-project/test-infra/prow/config.yaml -j ../../../../kyma-project/test-infra/prow/jobs -i preset-sa-gke-kyma-integration -e preset-sa-kyma-artifacts`
```

Comment:
``````suggestion
To use the prowjobparser, run it with appropriate parameters. See the example:

```bash
go run main.go \
  -c ../../../../kyma-project/test-infra/prow/config.yaml \
  -j ../../../../kyma-project/test-infra/prow/jobs \
  -i preset-sa-gke-kyma-integration \
  -e preset-sa-kyma-artifacts
```
``````